homeassistant/components/nest/camera_sdm.py

```diff
@@ -207,30 +203,18 @@ async def async_camera_image(
         self, width: int | None = None, height: int | None = None
     ) -> bytes | None:
         """Return bytes of camera image."""
-        if CameraEventImageTrait.NAME in self._device.traits:
-            # Returns the snapshot of the last event for ~30 seconds after the event
-            event_media: EventMedia | None = None
-            try:
-                event_media = (
-                    await self._device.event_media_manager.get_active_event_media()
-                )
-            except ApiException as err:
-                _LOGGER.debug("Failure while getting image for event: %s", err)
-            if event_media:
-                return event_media.media.contents
-        # Fetch still image from the live stream
-        stream_url = await self.stream_source()
-        if not stream_url:
-            if self.frontend_stream_type != STREAM_TYPE_WEB_RTC:
-                return None
-            # Nest Web RTC cams only have image previews for events, and not
-            # for "now" by design to save batter, and need a placeholder.
-            if not self._placeholder_image:
-                self._placeholder_image = await self.hass.async_add_executor_job(
-                    PLACEHOLDER.read_bytes
-                )
-            return self._placeholder_image
-        return await async_get_image(self.hass, stream_url, output_format=IMAGE_JPEG)
+        # Use the thumbnail from RTSP stream
+        stream = await self.async_create_stream()
+        if stream:
+            return await stream.async_get_image(width, height)
+        if self.frontend_stream_type != STREAM_TYPE_WEB_RTC:
+            return None
+        # Use a placeholder image for WebRTC cameras with no thumbnail
```

Comment:
Maybe move this comment up above line 210?

Reply:
Your comment triggered me to clean up a few other things, since the reason I ended up putting it here was because of the awkward frontend_stream_type check. Now, this is simplified so that if a camera is missing an RTSP stream it will show the placeholder, rather than only for rtsp cameras. That would be unexpected, but seemed simpler. Then, i cleaned up some of the test code to make the two image cases more clear.